tests/test_postgresql.py

```diff
@@ -860,7 +860,7 @@ def teardown_class(cls):
 
     @provide_metadata
     def _expect_default(self, c_expected, col, seq=None):
-        Table("t", self.metadata, col)
+        Table("t", self.metadata, col, schema="test_schema")
```

Comment:
it would be better to have both cases tested.

I think you could be better to have a parameter passed to the outside to enable /disable the schema. Then we could have separate tests that run or not the schema (if could also use `combinations()`)